wrappers/python/pyrs_device.cpp

```diff
@@ -40,6 +40,8 @@ void init_device(py::module &m) {
             std::stringstream ss;
             ss << "<" SNAME ".device: " << self.get_info(RS2_CAMERA_INFO_NAME)
                 << " (S/N: " << self.get_info(RS2_CAMERA_INFO_SERIAL_NUMBER)
+                << "  FW: " << self.get_info(RS2_CAMERA_INFO_FIRMWARE_VERSION)
```

Comment:
Please build the string dynamically -- it won't work if the device does not support a FW version or locked status (I just tried with the T265 and rang into this issue: no lock status)